frontend/src/pages/RoomPage/components/CollabContainer/components/Results.tsx

```diff
@@ -1,10 +1,16 @@
 import { Box, Heading, Text } from "@chakra-ui/react";
 
-const Results = () => {
+const Results = ({codeResult}: {codeResult:{ result: string, iserror: boolean}}) => {
   return (
     <Box padding={2}>
       <Heading>Results</Heading>
-      <Text>Results Details</Text>
+      {
+        // Font has to be monospaced to allow error messages to be formatted properly
+        // whitespace=pre preserves whitespaces
+        codeResult.iserror ?
+        <Text whiteSpace={"pre"} fontFamily='monospace' color='tomato' >{`${codeResult.result}`}</Text> :
```

Comment:
Can put conditional statement in the tag, i.e. `<Text ... color={codeResult.iserror ? 'tomato' : undefined} >`